src/rpc/blockchain.cpp

```diff
@@ -936,10 +936,10 @@ static RPCHelpMan getblock()
 
     int verbosity = 1;
     if (!request.params[1].isNull()) {
-        if(request.params[1].isNum())
-            verbosity = request.params[1].get_int();
-        else
+        if(request.params[1].isBool())
```

Comment:
```suggestion
        if (request.params[1].isBool()) {
```

please use the new style per the dev notes

Reply:
Added tests and updated the code to the new style